x-pack/plugins/actions/common/validate_email_addresses.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parseAddressList } from 'email-addresses';
+import { ValidatedEmail, InvalidEmailReason } from './types';
+import { hasMustacheTemplate } from './mustache_template';
+
+export interface ValidateEmailAddressesOptions {
+  allowMustache?: boolean;
```

Comment:
Following on from my previous comment --

I'd suggest 1. Renaming this to something like `dangerouslyAllowMustacheBypass` (so that future consumers won't _accidentally_ use this option), and 2. Adding a TSdoc comment here explaining what this option does.

Reply:
not a bad idea!